src/main/java/emissary/place/VersionPlace.java

```diff
@@ -58,12 +62,15 @@ public VersionPlace() throws IOException {
 
     private void configurePlace() {
         includeDate = configG.findBooleanEntry("INCLUDE_DATE", true);
+        useAbbrevHash = configG.findBooleanEntry("USE_ABBREV_HASH", false);
```

Comment:
Open to what others think but I would prefer this default to `true`.

Reply:
I think we use the full hash now? Do you propose changing to just use abbreviated instead in the RELEASE file? I'm good with either, I don't think it would break anything downstream.

Reply:
For a metadata field that will already have the version string, I'd vote for the abbreviation since it really just differentiates between SNAPSHOTs.  No need for the full 40-char string

Reply:
I would need to double check but I do not think we save the hash as a metadata field at all currently.  I had not considered how this interacts with the RELEASE file. 

But like @drivenflywheel said the full hash just seemed more then necessary. But this is configurable so it really does not matter.